configuration/pih/htmlforms/hiv/section-hiv-history.xml

```diff
@@ -809,9 +901,22 @@
                                 style="radio"/>
                         </p>
                     </div>
+                    <div>
+                        <obs id="contact-hiv-phone-{0}" conceptId="PIH:TELEPHONE NUMBER OF CONTACT"
+                             labelCode="pihcore.telephone"/>
```

Comment:
Instead of pihcore.telephone (not in the message property file), you can use this?

zl.registration.patient.contactPerson.phonenumber.label

Reply:
For the message codes I added them in messageProperties which is in the openmrs-config-pihemr project